bridge/client/assets/default-branding/app-config.json

```diff
@@ -1,6 +1,6 @@
 {
   "appTitle": "keptn",
-  "logoUrl": "assets/default-branding/logo.png",
+  "logoUrl": "assets/default-branding/logo.svg",
```

Comment:
At the moment the file ends with `png`. Could you set it to `png` again

Reply:
Damn rebase. Fixed it 👍🏻